main.tf

```diff
@@ -51,7 +51,7 @@ locals {
   }
 
   cluster_type_output_regional_zones = "${concat(google_container_cluster.primary.*.additional_zones, list(list()))}"
-  cluster_type_output_zonal_zones    = "${concat(google_container_cluster.zonal_primary.*.additional_zones, list(list()))}"
+  cluster_type_output_zonal_zones    = "${concat(slice(var.zones,1,length(var.zones)), list(list()))}"
```

Comment:
This is the relevant change

Reply:
For my own understanding, is the issue `google_container_cluster.zonal_primary.*.additional_zones` sometimes resolves to null instead of a list?

Reply:
> For my own understanding, is the issue google_container_cluster.zonal_primary.*.additional_zones sometimes resolves to null instead of a list?

Ah, I was able to clarify my understanding back in the bug report in #43 

```
Error: Error running plan: 1 error(s) occurred:

* module.gke.local.cluster_type_output_zonal_zones: local.cluster_type_output_zonal_zones: Resource 'google_container_cluster.zonal_primary' does not have attribute 'additional_zones' for variable 'google_container_cluster.zonal_primary.*.additional_zones'
```